src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -1555,7 +1555,7 @@ namespace nupic {
     UInt trueNeighborsMap1[8] = {0, 0, 1, 0, 1, 0, 0, 0};
     sp.getNeighbors1D_(column, dimensions, radius, wrapAround,
                           neighbors);
-    neighborsMap.clear();
+    neighborsMap.assign(numColumns, 0);
     for (auto & neighbor : neighbors) {
```

Comment:
clear() reduces this to zero entries, whereas this _seems_ to requires a zero'd array?